src/structures/gdt.rs

```diff
@@ -149,6 +168,25 @@ impl GlobalDescriptorTable {
         unsafe { lgdt(&self.pointer()) };
     }
 
+    /// Loads the GDT in the CPU using the `lgdt` instruction. This does **not** alter any of the
+    /// segment registers; you **must** (re)load them yourself using [the appropriate
+    /// functions](crate::instructions::segmentation):
+    /// [load_ss](crate::instructions::segmentation::load_ss),
+    /// [set_cs](crate::instructions::segmentation::set_cs).
+    ///
+    /// # Safety
+    ///
+    /// Unlike `load` this function will not impose a static lifetime constraint
+    /// this means its up to the user to ensure that there will be no modifications
+    /// after loading and that the GDT will live for as long as it's loaded.
+    ///
+    #[cfg(feature = "instructions")]
+    #[inline]
+    pub unsafe fn load_unchecked(&self) {
+        use crate::instructions::tables::lgdt;
+        lgdt(&self.pointer());
+    }
```

Comment:
We already have a similar method for interrupt descriptor tables that is named [`load_unsafe`](https://docs.rs/x86_64/0.12.3/x86_64/structures/idt/struct.InterruptDescriptorTable.html#method.load_unsafe). It might make sense to use the same naming convention here.